samples/python/winml_tutorial/.gitignore

```diff
@@ -1,3 +1,4 @@
 /pywinrt_output/build/
 /pywinrt_output/vc140.pdb
+/pywinrt_output/_pyrt.*.pyd
```

Comment:
I thought we had discussed not checking in projection output, but instead dropping it in another repo, gist, package, what have you.  it makes it harder to review large PRs with generated files.  thoughts?

Reply:
possibly just a dedicated branch (not master)?

Reply:
This is only for the sample. The source is schecked in so that folks who want to follow the sample don't have to build the pywinrt tool